front/lib/models.ts

```diff
@@ -10,7 +10,7 @@ import {
 
 const { FRONT_DATABASE_URI } = process.env;
 
-const front_sequelize = new Sequelize(FRONT_DATABASE_URI as string, {
+export const front_sequelize = new Sequelize(FRONT_DATABASE_URI as string, {
```

Comment:
not needed anymore. This was to leverage databse transactions but I reverted course.